Scripts/custom_hypr.lst

```diff
@@ -26,6 +26,8 @@ qt5-graphicaleffects                                   # for sddm theme effects
 hyprland                                               # wlroots-based wayland compositor
 dunst                                                  # notification daemon
 rofi-lbonn-wayland-git                                 # application launcher
+noto-fonts-emoji                                       # emoji
+rofi-emoji-git                                         # rofi emojis
```

Comment:
Hi Sadly, IIRC think the owner of the repo avoids extra deps on this repo

Reply:
umm ok so what can i do should i add the deps on any other repo?